ci_framework/roles/libvirt_manager/templates/kustomize_compute.yml.j2

```diff
@@ -0,0 +1,28 @@
+{%- set compute_range = range(0, compute_config.amount | default(1) | int) | list -%}"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+namespace: {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+patches:
+- target:
+    kind: OpenStackControlPlane
+  patch: |-
```

Comment:
I am not sure we need this file. The current install_yamls kustomization works on [this cr file](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplane.yaml) . If a particular path does not exist. The kustomization will does not work.

for example:
```
path: /spec/nodes/edpm-compute-2/ansibleHost
```
is not present in the CR file then it will fail.